### PR TITLE
GH2465: Add additional tool names for Roundhouse

### LINKED ISSUE #2465: Roundhouse dotnet tool does not run
### What You Are Seeing?
Using the Roundhouse .Net Core tool with `DotNetCoreTool` does not work as it says `No executable found matching command "dotnet-roundhouse`. This is probably as the Roundhouse .Net Core tool is installed as `rh` and the alias that is created in the cake folder is `rh.ex`.

### What is Expected?
The tool should run correctly.

### What version of Cake are you using?
0.32.1

### Are you running on a 32 or 64 bit system?
64

### What environment are you running on?  Windows? Linux? Mac?
Windows

### Are you running on a CI Server?  If so, which one?
No

### How Did You Get This To Happen? (Steps to Reproduce)
1. Create a new cake project.
2. Add `#module nuget:?package=Cake.DotNetTool.Module`
3. Run `build.ps1 --bootstrap`
4. Add `#tool dotnet:?package=dotnet-roundhouse`
5. Run `DotNetCoreTool("roundhouse");` from any cake task.

### PR DESCRIPTION
**Pull request by gep13**
Fixes #2465

### CODE REVIEW

File: src/Cake.Common/Tools/Roundhouse/RoundhouseRunner.cs [hunk 1]
```diff
@@ -156,7 +156,7 @@ protected override string GetToolName()
         /// <returns>The tool executable name.</returns>
         protected override IEnumerable<string> GetToolExecutableNames()
         {
-            return new[] { "rh.exe" };
+            return new[] { "rh.exe", "dotnet-roundhouse", "dotnet-roundhouse.exe" };
```
Comment (Meir017) [inline src/Cake.Common/Tools/Roundhouse/RoundhouseRunner.cs]: use expression body?

Comment (gep13) [reply src/Cake.Common/Tools/Roundhouse/RoundhouseRunner.cs]: Not sure if it is justified.  What is here is in keeping with other PR's for similar changes:

https://github.com/cake-build/cake/pull/2418
https://github.com/cake-build/cake/pull/2388

**Comment by devlead** [comment]
@gep13 your changes have been merged, thanks for your contribution 👍

**Comment by gep13** [comment]
@devlead sorry, just seeing this now. Thanks for pulling this in!

**Pull request merged**